cmd/bb_storage/main.go

```diff
@@ -15,8 +15,16 @@ import (
 	"github.com/buildbarn/bb-storage/pkg/cas"
 	"github.com/buildbarn/bb-storage/pkg/util"
 	"github.com/grpc-ecosystem/go-grpc-prometheus"
+	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 
+	prometheus_exporter "contrib.go.opencensus.io/exporter/prometheus"
```

Comment:
The other of imports has so far been to sort alphabetically by second-level domain. First comes `github.com`, followed by `golang.org`. Could we put `opencensus.io` after that?